content/docs/avalanche-l1s/maintain/remove-validator-l1.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Remove Validator
+description:  Learn how to remove a validator from an Avalanche L1.
+---
+
+### Remove a Validator from an Avalanche L1
+
+```bash
+avalanche blockchin removeValidator <blockchainName>
+```
+
+#### Choose the Network
```

Comment:
markdown formatting usually includes blank lines between headings and code blocks